apix-impl/src/test/java/eu/xenit/apix/tests/search/FtsNodeVisitorTest.java

```diff
@@ -0,0 +1,108 @@
+package eu.xenit.apix.tests.search;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.mock;
```

Comment:
@driesdeproost You don't seem to be using mocks anymore so you can remove this import